presto-product-tests/README.md

```diff
@@ -1,125 +1,106 @@
 # Presto product tests
 
-## Test configuration
+## Installing dependencies
 
-Test assume that you have hadoop and Presto cluster running. 
+*Running product test requires at least 4GB free memory to be available.*
 
-To make it work you need either:
- - define environment variables for hadoop ${HADOOP_MASTER} and Presto ${PRESTO_MASTER} with their IP numbers
- - create test-configuration-local.yaml with following (example content):
+#### On GNU/Linux
+* [```docker >= 1.10```](https://docs.docker.com/installation/#installation)
 
-```
-databases:
-  hive:
-    host: 172.16.2.10
-
-  presto:
-    host: 192.168.205.1
-```
-
-## Running tests
+    ```
+    wget -qO- https://get.docker.com/ | sh
+    ```
 
-Product tests are not run by default. To start them use run following command:
+* [```docker-compose >= 1.60```](https://docs.docker.com/compose/install/)
 
-```
-java -jar target/presto-product-tests-*-executable.jar --config-local tempto-configuration-local.yaml
-```
+    ```
+    pip install docker-compose
+    ```
 
-## Running tests with Using preconfigured docker based clusters
+#### On Mac OS X
 
-In case you do not have an access to hadoop cluster or do not want to spent your time on configuration, this section
-describes how to run product tests with preconfigured docker based clusters.
+* [```docker-toolbox >= 1.10```](https://www.docker.com/products/docker-toolbox)
 
-### Prerequisites
+On Mac OS X installing docker-toolbox gives access to preconfigured bash environment
+with ```docker``` and ```docker-compose``` available. To start this bash environment
+select "Docker Quickstart Terminal" from Launchpad. Note that all commands given in
```

Comment:
1. What choice? There is nothing to choose from in this docs.
2. Launchpad: https://en.wikipedia.org/wiki/Launchpad_(OS_X). But I will change it to sth like "you can find it in Applications/Docker".
3. There is no choice what to install while installing ```docker-toolbox``` there are just shortcuts to "Docker Quickstart Terminal" and "Kinematic" in last step of installer. Both "Docker Quickstart Terminal" and "Kinematic" are installed anyway.
4. Ditto about "how to use" vs. "how it works".